### DISCUSSION TITLE
20240802版本，安装passwall插件报错

### DISCUSSION THREAD

**Discussion by emlszhou** [istoreos/istoreos#1771] [Q&A]
新装的iStoreOS系统
在安装passwall和passwall2插件都报错，不知道什么原因，希望大神帮一下忙
报错信息如下

![image](https://github.com/user-attachments/assets/aee2e98b-b26c-44e0-907b-9423787475d8)
![image](https://github.com/user-attachments/assets/139a74fb-82ba-49b2-a819-982da75e5905)

**Comment by IWIAI** [comment]
前置依赖没装，到这个仓库里下载依赖包先安装上，https://github.com/kenzok8/small/releases

**Reply by Fannie0124** [reply]
大佬安装哪个啊  看不懂

**Comment by jjm2473** [comment]
你确定不是看错架构了？例如把x86的软件包安装到arm机器上了
